analysis/scripts/2233-power-calculations.Rmd

```diff
@@ -214,40 +226,49 @@ simulation_01 <- function(
   treatment = complete_ra(N = in_population_size,prob = in_prop_assigned_treatment),
   outcome_0 = outcome_0,
   outcome_1 = case_when(
-    effect*treatment + outcome_0 < 0 ~ 0,
-    effect*treatment + outcome_0 >= 0 ~ effect*treatment + outcome_0
+    effect + outcome_0 < 0 ~ 0,
```

Comment:
This is the key change that took me too long to notice.

Reply:
Great. Thanks. That makes sense. I think that I thought I was calculating the obs effect here, but this makes sense.